enso/visualize/facets.py

```diff
@@ -40,6 +40,9 @@ def visualize(
             ylim=y_limits,
             palette='deep'
         )
-        grid = grid.map(sns.pointplot, x_axis, y_axis).add_legend()
+        grid = grid.map(sns.pointplot, x_axis, y_axis)
+        colors = sns.color_palette('deep').as_hex()[:len(grid.hue_names)]
+        handles = [patches.Patch(color=col, label=label) for col, label in zip(colors, grid.hue_names)]
+        plt.legend(handles=handles)
```

Comment:
I assume this makes the legends actually work as opposed to their previous lack of functionality. Awesome

Reply:
yup!